bogo/config.json.in

```diff
@@ -17,7 +17,6 @@
     "CheckLeafCurve": "",
     "SendWarningAlerts-*": "",
     "Peek-*": "",
-    "EchoTLS13CompatibilitySessionID": "",
```

Comment:
It looks like this ignore was added way back in 2018 (https://github.com/rustls/rustls/commit/32eeec61d3bd63fc8eafca39fb6e5807c448076f) without any special mention. 

I was looking at the test case for other reasons and couldn't think of a reason why we shouldn't add the coverage. WDYT?